contracts/tokens/ERC721implementation.sol

```diff
@@ -230,6 +230,7 @@ contract ERC721implementation is Ownable, ERC721, ERC165implementation {
     external
   {
     require(_operator != address(0));
+    require(ownerToNFTokenCount[msg.sender] > 0);
```

Comment:
Why is this necessary? It costs additional gas for token owners and does no harm if someone else uses it except that they have to pay gas.. And there could as well be a use case where someone gives operator rights even before he gets a token..

Reply:
@MoMannn everything is in the commit message. It's a proposal. wdyt?

Reply:
I would leave it as it is.. Like I mentioned someone could give operator rights before he owns tokens and even otherwise it does no damage but it uses gas..